linux/network_device_ops/pvrdma_client_network_device_ops.yml

```diff
@@ -10,7 +10,7 @@
         vm_name: "{{ pvrdma_client_vm_name }}"
         current_test_log_folder: "{{ pvrdma_client_test_log_folder }}"
         pvrdma_client_vm_ready: false
```

Comment:
`pvrdma_client_vm_ready` seems not used.

Reply:
It's used in rescue block for collecting logs.